src/block/Anvil.php

```diff
@@ -99,4 +99,12 @@ public function place(BlockTransaction $tx, Item $item, Block $blockReplace, Blo
 	public function tickFalling() : ?Block{
 		return null;
 	}
+
+	public function getFallDamagePerDistance() : float{
```

Comment:
I think this should be named `getFallDamagePerBlock` because the distance measurement is on a per-block basis. Using the word "Distance" here indicates something other than minecraft's blocks are being user for measurement.

Alternatively, this function name can be shortened to omit the "PerDistance" part with `getFallDamage()` though I personally prefer the verbosity of the former option.

Reply:
getFallDamagePerBlock() would be my preference. The latter is too ambiguous.